docs/en/wallet-provider-endpoint.rst

```diff
@@ -394,7 +366,7 @@ The body of the Wallet Attestation SD-JWT contains the following claims:
       - REQUIRED. JSON object, containing the public part of an asymmetric key pair owned by the Wallet Instance.
       - :rfc:`7800`.
     * - **vct**
-      - REQUIRED. Credential type value MUST be an HTTPS URL String and it MUST be set to ``wallet.atestation.example/v1.0``.
+      - REQUIRED. Credential type value MUST be an HTTPS URL String and it MUST be set to ``https://trust-registry.it-wallet.example.it/v1.0/WalletAttestation``.
```

Comment:
This is also an example. It is necessary to regulate the fact that the VCT must be of the type `{url_trust_anchor}/{version}/WalletAttestation` by defining what the URL and version must be.